.github/workflows/build-image.yml

```
@@ -280,23 +280,40 @@ jobs:
         working-directory: /tmp/digests
         run: |
           SHORT_SHA=${GITHUB_SHA:0:7}
```

We talked about this in slack: https://redhat-internal.slack.com/archives/C09HYSPUTKR/p1761844074006769
```suggestion
          SHORT_SHA=${{ github.event.pull_request.head.sha || github.sha }}
```